shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -1132,3 +1132,278 @@ rowArchivalVisibilityClause
 defaultCollationClause
     : DEFAULT_COLLATION EQ_ (collationName | NONE)
     ;
+
+alterDatabase
+    : ALTER DATABASE databaseName?
```

Comment:
`database_clause` is not complete 

![image](https://user-images.githubusercontent.com/66914151/116558217-7ea80700-a8ff-11eb-9adf-3da4733a4265.png)

Reply:
https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/img_text/alter_database.html

Reply:
in #10136, 
>  tristaZero said
Oracle Ref -> https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlqr/SQL-Statements.html


we just support oracle version 19.

Reply:
@Liangda-w can you give me ref url.

Reply:
@Liangda-w OK, I have found my problem, thanks.^_^